src/components/navigation/AppNavigationBoard.vue

```diff
@@ -272,9 +272,7 @@ export default {
 							.then(() => {
 								this.loading = false
 								this.deleted = true
-								this.undoTimeoutHandle = setTimeout(() => {
-									this.$store.dispatch('removeBoard', this.board)
-								}, 7000)
+								this.$store.dispatch('removeBoard', this.board)
```

Comment:
I guess this was for making testing easier, but please revert this change as the undo option should still be available.

Reply:
@juliushaertl When we delete a board, we need to wait for 7s which is too long? Is there any reason for that?

Reply:
UX-wise the idea is that delete is a destructive action and since we currently have nothing like a trashbin for deleted boards, the timeout will allow you to still undo this for some seconds. Lets not change that for now, but we could look into the deletion as an enhancement in the future.